airbyte-cdk/bulk/toolkits/load-azure-blob-storage/src/main/kotlin/io/airbyte/cdk/load/file/azureBlobStorage/AzureBlobStorageClient.kt

```diff
@@ -23,12 +26,15 @@ import kotlinx.coroutines.flow.flow
  */
 const val GENERATION_ID_METADATA_KEY_OVERRIDE = "ab_generation_id"
 
+private const val DELETE_BATCH_SIZE = 256
+
 /** Represents a single blob in Azure. */
 data class AzureBlob(
     override val key: String,
     override val storageConfig: AzureBlobStorageClientConfiguration
 ) : RemoteObject<AzureBlobStorageClientConfiguration>
 
+@SuppressFBWarnings("NP_NONNULL_PARAM_VIOLATION")
```

Comment:
Suspend stuff?

Reply:
@tryangul It's a bug in spotbugs with Kotlin.  I don't think it has to do with suspend as much as it is something with it not being able to determine the nullability of the parameter.  We suppress this in a few other places in the CDK, so this isn't the first time we have run into this.